common/src/main/scala/hmda/model/modifiedlar/ModifiedLoanApplicationRegister.scala

```diff
@@ -102,6 +102,8 @@ case class ModifiedLoanApplicationRegister(
 ) extends PipeDelimited
     with ColumnDataFormatter {
 
+  def header: String = """Record Identifier|Legal Entity Identifier (LEI)|Loan Type|Loan Purpose|Preapproval|Construction Method|Occupancy Type|Loan Amount|Action Taken|State|County|Census Tract|Ethnicity of Applicant or Borrower: 1|Ethnicity of Applicant or Borrower: 2|Ethnicity of Applicant or Borrower: 3|Ethnicity of Applicant or Borrower: 4|Ethnicity of Applicant or Borrower: 5|Ethnicity of Co-Applicant or Co-Borrower: 1|Ethnicity of Co-Applicant or Co-Borrower: 2|Ethnicity of Co-Applicant or Co-Borrower: 3|Ethnicity of Co-Applicant or Co-Borrower: 4|Ethnicity of Co-Applicant or Co-Borrower: 5|Ethnicity of Applicant or Borrower Collected on the Basis of Visual Observation or Surname|Ethnicity of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname|Race of Applicant or Borrower: 1|Race of Applicant or Borrower: 2|Race of Applicant or Borrower: 3|Race of Applicant or Borrower: 4|Race of Applicant or Borrower: 5|Race of Co-Applicant or Co-Borrower: 1|Race of Co-Applicant or Co-Borrower: 2|Race of Co-Applicant or Co-Borrower: 3|Race of Co-Applicant or Co-Borrower: 4|Race of Co-Applicant or Co-Borrower: 5|Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname|Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname|Sex of Applicant or Borrower|Sex of Co-Applicant or Co-Borrower|Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname|Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname|Age of Applicant or Borrower|Age of Applicant >= 62|Age of Co-Applicant or Co-Borrower|Age of Co-Applicant >= 62|Income|Type of Purchaser|Rate Spread|HOEPA Status|Lien Status|Applicant or Borrower - Name and Version of Credit Scoring Model|Co-Applicant or Co-Borrower - Name and Version of Credit Scoring Model|Reason for Denial: 1|Reason for Denial: 2|Reason for Denial: 3|Reason for Denial: 4|Total Loan Costs|Total Points and Fees|Origination Charges|Discount Points|Lender Credits|Interest Rate|Prepayment Penalty Term|Debt-to-Income Ratio|Combined Loan-to-Value Ratio|Loan Term|Introductory Rate Period|Balloon Payment|Interest-Only Payments|Negative Amortization|Other Non-Amortizing Features|Property Value|Manufactured Home Secured Property Type|Manufactured Home Land Property Interest|Total Units|Multifamily Affordable Units|Submission of Application|Initially Payable to Your Institution|Automated Underwriting System: 1|Automated Underwriting System: 2|Automated Underwriting System: 3|Automated Underwriting System: 4|Automated Underwriting System: 5|Reverse Mortgage|Open-End Line of Credit|Business or Commercial Purpose"""
```

Comment:
@Kibrael  can you confirm we've dotted all i's and crossed all t's?

Reply:
The header has all the correct fields in the correct order.